packages/common/src/utils/nft.utils.ts

```diff
@@ -8,10 +8,9 @@
 import { LambdaClient, InvokeCommand, InvokeCommandOutput } from '@aws-sdk/client-lambda';
 import { addAlert } from './alerting.utils';
 import { Severity } from '../types';
-// @ts-ignore
 import { Network, constants, CreateTokenTransaction, helpersUtils } from '@hathor/wallet-lib';
-// @ts-ignore
-import type { HistoryTransaction } from '@hathor/wallet-lib';
+// FIXME: import from lib path on HathorLib
+import type { HistoryTransaction } from '@hathor/wallet-lib/lib/models/types';
```

Comment:
Why can't we do it?

Reply:
The `HistoryTransaction` is not exported on the lib index but it is used here so it needs to be imported from the lib internal path.
Ideally, later we will export this type from the lib index and move this to import from `@hathor/wallet-lib`.